src/main/java/com/example/spring/services/UserDetailsServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.example.spring.services;
```

Comment:
if you have security package, it's better to move CustomUserDetailsService (rename) there